docker/dyno/app/templates/index.html

```diff
@@ -47,76 +46,103 @@
+                  <p class="card-text"><span class="fw-bold">Er:</span> Error rate</p>
+                </div>
+              </div>
+            </div>
 
+            <div class="col-4">
+              <div class="card">
+                <h5 class="card-header bg-danger text-white">Network conditions</h5>
+                <div class="card-body">
+                  <p class="card-text"><span class="fw-bold">L:</span> Latency</p>
+                  <p class="card-text"><span class="fw-bold">J:</span> Jitter</p>
+                  <p class="card-text"><span class="fw-bold">B:</span> Bandwidth</p>
+                  <p class="card-text"><span class="fw-bold">SC:</span> Slow close</p>
+                  <p class="card-text"><span class="fw-bold">Sas:</span> Slicer [Avg size]</p>
+                  <p class="card-text"><span class="fw-bold">Sd:</span> Slicer [Delay]</p>
+                </div>
+              </div>
+            </div>
 
-                   
-  <div id="load-controls">
-    <form action="#">
-      <fieldset>
-        <label for="scenarios">Select a load generation scenario</label>
-        <p></p>
-        <select name="scenarios" id="scenarios_selector">
-          <option selected="selected">Default</option>
-        </select>
-        <hr>
-        <div id="load-stats" style="width: 40px">
-            <table id="get_stats" style="float:left;"></table>
-            <table id="post_stats" style="float:left;"></table>
+            <div class="col-4">
+              <div class="card">
+                <h5 class="card-header bg-primary text-white">Operating system</h5>
+                <div class="card-body">
+                  <p class="card-text"><span class="fw-bold">CPU:</span> Processor quota</p>
+                  <p class="card-text"><span class="fw-bold">Mem:</span> Container memory</p>
+                </div>
+              </div>
+            </div>
+          </div>
         </div>
+        <div class="modal-footer">
+          <button type="button" class="btn btn-primary" data-bs-dismiss="modal">Close</button>
+        </div>
+      </div>
+    </div>
+  </div>
 
+  <div class="container-fluid">
+    <div class="row">
+      <div id="load-controls" class="col-sm-2">
+          <label for="scenarios" class="fw-bold">Select a load generation scenario:</label>
+          <p></p>
+          <select name="scenarios" id="scenarios_selector" class="form-select" aria-label="Load generation examples selector">
+            <option selected="selected">Default</option>
+          </select>
+          <hr>
+          <div id="load-stats">
+            <div id="get_stats" class="card" id="get-stats-card"></div>
+            <div id="post_stats" class="card" id="post-stats-card"></div>
+          </div>
+      </div>
+      <div class="col-sm-10">
+        <div class="row" id="proxy-container"></div>
+      </div>
+    </div>
+    <div class="row" id="load-container"></div>
```

Comment:
I do not know what is the purpose of this div-container. Is it still needed?
```suggestion
    <div class="row" id="load-container"></div>
```

Reply:
This is necessary. Please don't remove it. It contains the `load-controls` div.